guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc

```diff
@@ -33,24 +35,58 @@ the upgrade to proceed.
 +
 For information on backups, see {AdministeringDocURL}backing-up-satellite-server-and-capsule-server[Backing Up {ProjectServer} and {SmartProxyServer}] in the _Administering {ProjectName} {ProjectVersionPrevious}_ guide.
 
-. Optional: If you made manual edits to DNS or DHCP configuration in the `/etc/zones.conf` or `/etc/dhcp/dhcpd.conf` files, back up the configuration files because the installer only supports one domain or subnet, and therefore restoring changes from these backups might be required.
-
-. Optional: If you made manual edits to DNS or DHCP configuration files and do not want to overwrite the changes, enter the following command:
+ifdef::katello[]
+. Update repositories for el7
 +
 [options="nowrap" subs="attributes"]
 ----
-# {foreman-installer} --foreman-proxy-dns-managed=false \
---foreman-proxy-dhcp-managed=false
+# yum update -y http://yum.theforeman.org/katello/{KatelloVersion}/katello/el7/x86_64/katello-repos-latest.rpm
+# yum update -y https://yum.theforeman.org/releases/{ProjectVersion}/el7/x86_64/foreman-release.rpm
```

Comment:
While not today needed, I think this needs to be a single command.
```suggestion
# yum update -y http://yum.theforeman.org/katello/{KatelloVersion}/katello/el7/x86_64/katello-repos-latest.rpm https://yum.theforeman.org/releases/{ProjectVersion}/el7/x86_64/foreman-release.rpm
```

The reason is that `katello-repos` requires `foreman-release`. That's why I think you should update `foreman-release` first.

I also think that we should actually update the spec to also require specific releases so there can't be a version mismatch. We did this with katello itself, but if you get it earlier, it's even better.

That means updating this line:
https://github.com/theforeman/foreman-packaging/blob/f2421fa4d7cbeabf8f7974a02525c20fbd7e9c41/packages/katello/katello-repos/katello-repos.spec#L24

If we already have the documentation ready for that, we can apply it any time we want.

(same below for EL8)

Reply:
sounds good to me